src/components/Auth/PinSetup/PinPad/Header/styles.js

```diff
@@ -9,89 +8,73 @@ import {
   helpers,
 } from 'components/shared/Styleguide';
 
-const titleStyles = {
-  flex: 1,
-  flexWrap: 'wrap',
-  fontFamily: fonts.helveticaNeue.bold,
-  fontSize: fontSizes.secondaryTitle,
-  textAlign: 'center',
-};
-
-const subtitleStyles = {
-  fontSize: fontSizes.default,
-  lineHeight: helpers.calculateLineHeight(fontSizes.default),
-  paddingTop: 10,
-  position: 'absolute',
-  textAlign: 'center',
-  top: '100%',
-  width: helpers.vw - 48,
-};
-
-const dotStyles = {
+export const dotStyles = {
   borderRadius: 10,
   height: 20,
   marginLeft: 9,
   marginRight: 9,
   width: 20,
 };
 
-export default StyleSheet.create({
-  container: {
-    flex: 2,
-  },
-  content: {
-    alignItems: 'center',
-    flex: 2,
-    justifyContent: 'space-evenly',
-    flexDirection: 'column',
-  },
-  textContainer: {
-    alignItems: 'stretch',
-    flex: 0,
-    flexDirection: 'column',
-    justifyContent: 'center',
-    marginLeft: 24,
-    marginRight: 24,
-  },
-  titleContainer: {
-    flexDirection: 'row',
-    flexShrink: 0,
-    width: helpers.vw - 48,
-  },
-  title: {
-    ...titleStyles,
-    color: colors.textDefault,
-  },
-  subtitle: {
-    ...subtitleStyles,
-    color: colors.textSecondary,
-  },
-  errorTitle: {
-    ...titleStyles,
-    color: colors.error,
-  },
-  errorSubtitle: {
-    ...subtitleStyles,
-    color: colors.error,
-  },
-  dotsContainer: {
-    alignItems: 'flex-start',
-    flex: 0,
-    flexDirection: 'row',
-    justifyContent: 'center',
-  },
-  dot: dotStyles,
-  emptyDot: {
-    ...dotStyles,
-    borderColor: '#BEBEBE',
-    borderWidth: 1,
-  },
-  errorDot: {
-    ...dotStyles,
-    borderColor: colors.error,
-    borderWidth: 1,
-  },
-});
+export const Container = styled.View`
+  flex: 2;
+`;
+
+export const Content = styled.View`
+  align-items: center;
+  flex: 2;
+  justify-content: space-evenly;
+  flex-direction: column;
+`;
+
+export const TextContainer = styled.View`
+  align-items: stretch;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 24;
+  margin-right: 24;
+`;
+
+export const TitleContainer = styled.View`
+  flex-direction: row;
+  flex-shrink: 0;
+  width: ${helpers.vw - 48};
+`;
+
+export const Title = styled.Text`
+  flex: 1;
+  flex-wrap: wrap;
+  font-family: ${fonts.helveticaNeue.bold};
+  font-size: ${fontSizes.secondaryTitle};
+  text-align: center;
+  color: ${props => (props.showError ? colors.error : colors.textDefault)};
+`;
+
+export const Subtitle = styled.Text`
+  font-size: ${fontSizes.default};
+  line-height: ${helpers.calculateLineHeight(fontSizes.default)};
+  padding-top: 10;
+  position: absolute;
+  text-align: center;
+  top: 100%;
+  width: ${helpers.vw - 48};
+  color: ${props => (props.showError ? colors.error : colors.textSecondary)};
```

Comment:
This will cause "unknown prop `showError` on <Text>"  like warnings. We'll need to workaround this using props filtering like this https://github.com/styled-components/styled-components/issues/135